datahub-web-react/src/app/entity/chart/ChartEntity.tsx

```diff
@@ -168,6 +168,7 @@ export class ChartEntity implements Entity<Chart> {
         return (
             <ChartPreview
                 urn={data.urn}
+                subType={data.subTypes?.typeNames?.[0]}
```

Comment:
Did we already do charts?

Reply:
yeah charts are done, this is just a fix for a missing prop